### DISCUSSION TITLE
Kopiowanie wykazu

### DISCUSSION THREAD

**Discussion by UniqueTM2** [TomaszGasior/RadioLista-v3#167] [3. Inne Other]
Mam jeszcze jedno pytanie. Potrzebuje testowo złożyć dwa wykazy ze sobą, ale klepanie około 800 pozycji będzie pracochłonne, wiec pomyślałem nad sklonowaniem jednego z nich i dopisywaniu tylko ewentualnie nowych danych z drugiego, ułatwiłoby mi to prace, a przede wszystkim straciłbym mniej cennego nocnego czasu :)

Czy jest technicznie taka możliwość?

**Comment by TomaszGasior** [comment]
Żebyśmy mogli załatwić taką sprawę, muszę wpierw zrozumieć twoją intencję.

- A: Czy chcesz jedynie rozwiązać jakiś jeden konkretny, jednorazowy problem?
- B: Czy chcesz, żeby RadioLista posiadała taką funkcję w standardzie dla każdego na stałe?

W przypadku wariantu A mogę spróbować dokonać manualnej interwencji do bazy danych, żeby coś dla ciebie skopiować, jeśli będzie to możliwe. W takim przypadku możemy kontynuować rozmowę tutaj lub mailowo, jak ci pasuje, i możesz mi napisać co dokąd kopiujemy, a ja ocenię, na ile to jest możliwe.

W przypadku wariantu B napotykamy na problem podobny jak w dyskusji #153 — żeby zaimplementować funkcję w systemie, musi ona mieć jakiś sens i musisz to uargumentować: podać różne przykłady użycia takiej funkcji. Nie ma sensu poświęcać masy energii do implementacji opcji, z której nikt nie będzie korzystał. Żeby mieć pewność, że taka funkcja ma sens, musimy o tym po prostu więcej pogadać.

**Comment by UniqueTM2** [comment]
Witaj. 
Potrzebuje zrobić to jednorazowo, po prostu chciałbym, aby jeden z wykazów zwyczajnie sklonować jako drugi, a ja później bede go dalej edytował

**Reply by TomaszGasior** [reply]
Jasne, podaj szczegóły.

**Comment by UniqueTM2** [comment]
(usunięte)
skopiować do:
(usunięte)

**Reply by TomaszGasior** [reply]
Skopiowane.

**Comment by UniqueTM2** [comment]
Widze właśnie, wyglada, że sie idealnie udało. Dziekuje Ci Tomku za pomoc :)
